google/cloud/security/scanner/audit/fw_rules_engine.py

```diff
@@ -106,9 +106,8 @@ def __init__(self,
         self.org_res_rel_dao = org_resource_rel_dao.OrgResourceRelDao(
             global_configs)
         self.snapshot_timestamp = None
```

Comment:
Please remove the previous self.snapshot_timestamp = None line.

Reply:
Done.